doc/doc-support/package.nix

```
@@ -96,15 +95,14 @@ stdenvNoCC.mkDerivation (
 
       dest="$out/share/doc/nixpkgs"
       mkdir -p "$(dirname "$dest")"
-      mv out "$dest"
-      mv "$dest/index.html" "$dest/manual.html"
+      cp -r out "$dest"
```

```suggestion
      mv out "$dest"
      cp "$dest/index.html" "$dest/manual.html"
```

I think we should really just do this here